test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class SessionsControllerTest < ActionDispatch::IntegrationTest
+  test 'should get new' do
```

Comment:
why not use rspec?

Reply:
These are some auto-generated tests. Should I modify them?